sccp/figures/common.py

```diff
@@ -255,7 +268,7 @@ def plotDrugUMAP(drugs, decomp, totaldrugs, points, axs):
     return
 
 
-def plotCmpUMAP(cellState, cmp, factors, pf2Points, allP, ax):
+def plotCmpUMAP(cmp, factors, pf2Points, allP, ax):
```

Comment:
I think if we're using proj x B here we don't need a cellState input (?)

Reply:
Thanks for the fix!